include/cantera/oneD/StFlow.h

```diff
@@ -325,25 +312,133 @@ class StFlow : public Domain1D
+     *
+     * This model uses the optically thin limit and the gray-gas approximation to
+     * simply calculate a volume specified heat flux out of the Planck absorption
+     * coefficients, the boundary emissivities and the temperature. Polynomial lines
+     * calculate the species Planck coefficients for H2O and CO2. The data for the
+     * lines are taken from the RADCAL program @cite RADCAL.
+     * The coefficients for the polynomials are taken from
+     * [TNF Workshop](https://tnfworkshop.org/radiation/) material.
+     */
+    void computeRadiation(double* x, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the continuity equation residual.
+     *
+     * This function calculates the residual of the continuity equation
+     * @f[
+     *     \frac{d(\rho u)}{dz} + 2\rho V = 0
+     * @f]
+     *
+     * The continuity equation propagates information from right-to-left.
+     * The @f$ \rho u @f$ at point 0 is dependent on @f$ \rho u @f$ at point 1 but not
+     * on @f$ \dot{m} @f$ from the inlet. The default value for the continuity equation
+     * is zero velocity (@f$ u @f$) at the left and right boundary.
+     */
+    virtual void evalContinuity(double* x, double* rsd, int* diag,
+                                double rdt, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the momentum equation residual.
+     *
+     * The function calculates the radial momentum equation defined as
+     * @f[
+     *    \rho \frac{dV}{dt} + \rho u \frac{dV}{dz} + \rho V^2 =
+     *    \frac{d(\mu \frac{dV}{dz})}{dz} - \lambda
+     * @f]
+     *
+     * The radial momentum equation incorporates terms for temporal and spatial
+     * variations of radial velocity (@f$ V @f$). The default boundary condition is zero
+     * radial velocity (@f$ V @f$) at the left and right boundary.
+     */
+    virtual void evalMomentum(double* x, double* rsd, int* diag,
                               double rdt, size_t jmin, size_t jmax);
 
+    /**
+     * Evaluate the lambda equation residual.
+     *
+     * The function calculates the lambda equation as
+     * @f[
+     *    \frac{d\lambda}{dz} = 0
+     * @f]
+     *
+     * The lambda equation serves as an eigenvalue that allows the momentum
+     * equation and continuity equations to be simultaneously satisfied. The lambda
+     * equation propgates information from left-to-right. The default
+     * boundary condition is zero (@f$ \lambda @f$) at the left and zero flux at
+     * the right boundary.
+     */
+    virtual void evalLambda(double* x, double* rsd, int* diag,
+                            double rdt, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the energy equation residual.
+     *
+     * The function calculates the energy equation:
+     * @f[
+     *    \rho c_p \frac{dT}{dt} + \rho c_p u \frac{dT}{dz} =
+     *    \frac{d(k \frac{dT}{dz})}{dz} - \sum_k (\omega_k h_{k_{\text{ref}}}) -
+     *    \sum_k \left( \frac{J_k c_{p_k}}{M_k} \frac{dT}{dz} \right)
```

Comment:
Again, no time derivative. See https://github.com/Cantera/cantera-website/blob/73af4ef147ed7782df7da05ebf2a0a7a50ae0753/pages/science/flames.rst?plain=1#L63-L66

Reply:
I believe those equation forms were from the original docstrings in the function. I assumed that the ones with the time derivative were ones where diag=1 and the rdt quantity was used. Should we eliminate the time derivative from the docstring?

Reply:
Yes. The 'time derivative' is a pseudo time-step in the solver. It's not a true transient simulation.